### PR TITLE
default spring naming strategies

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by geoand** [comment]
We discussed this and will probably make some changes

**Comment by aureamunoz** [comment]
could you, please,  take a look on last changes @geoand ? don't see the ci actions, it is work in progress, I pushed by mistake

File: pom.xml [hunk 1]
```diff
@@ -1,151 +1,128 @@
...
-  <developers>
-    <developer>
-      <id>jboss.org</id>
-      <name>JBoss.org Community</name>
-      <organization>JBoss.org</organization>
-      <organizationUrl>http://www.jboss.org</organizationUrl>
-    </developer>
-  </developers>
+    <name>Spring Boot dependencies for Quarkus Spring Spring Boot Properties extension</name>
+    <description>The minimum dependencies to reduce the footprint of Quarkus applications using the Spring Spring Boot
```
Comment (geoand) [inline pom.xml]: This would have to change the proper description added to each module

Comment (aureamunoz) [reply pom.xml]: :face_palm:

Comment (aureamunoz) [reply pom.xml]: :woman_facepalming:

**Comment by aureamunoz** [comment]
What do you think @geoand?

**Review by geoand — APPROVED** [review]
LGTM

**Comment by gsmet** [comment]
Two comments on this:
- never ever mix reformatting and changes in behavior. It makes things very hard when you look at your history. Agreed, this project is a bit simplistic but even there I wouldn't do it.
- now, there is the question of if they should be the default when using the Spring Data JPA compatibility layer or not? It would break the compatibility but would be more Spring-like. And I'm pretty sure people will miss it altogether if they aren't the default.

**Comment by geoand** [comment]
@gsmet yes, the idea is that we will add this dependency to the Spring Data JPA extension

**Comment by gsmet** [comment]
But does that make them the default ones used by ORM?

I mean having them around is different from making them the default (and breaking backward compatibility)

**Comment by geoand** [comment]
No, they are not the defaults in Spring Boot either. They just exist so users can select them by specifying some configuration.

The plan is that if show the corresponding quarkus configuration property if the user tries to use the Spring Boot one

**Comment by gsmet** [comment]
OK, I thought they were the default considering people were complaining about us being inconsistent.

**Comment by geoand** [comment]
Nah, it's just about adding the functionality (someone did ask for it)

**Comment by geoand** [comment]
As pointed out by @aureamunoz and @gsmet, someone did complain about this and brought forth the fact that Spring Boot **does** use these strategies. 
So I stand corrected

**Comment by aureamunoz** [comment]
So, can we merge and release this one?

**Comment by geoand** [comment]
Have you tested it locally with Quarkus (including native mode)?

**Comment by aureamunoz** [comment]
yes, I have tested and also in native mode. but I need to remove the modifications because finally we have decided to not do it, so once I will test the code, I will release this one

**Comment by aureamunoz** [comment]
Can I release this one as 2.2.Alpha1? As a reminder I did a mistake and release a [`2.2.Final`](https://repo.maven.apache.org/maven2/io/quarkus/quarkus-spring-boot-api/) version recently so I wonder about what should be the version to use now. WDYT @geoand @gsmet ?

**Comment by geoand** [comment]
Yes, please release `2.2.Alpha1` and use that to open a pull request and make sure everything is OK. But before merging, we will need a `2.2.SR1` (since this change is intended to be backported to `1.3.1.Final` and we can't have `Alpha` versions in there).

**Pull request merged**